docs-src/go/how-to-configure-max-concurrent-sessions.md

```diff
@@ -0,0 +1,26 @@
+---
+id: how-to-configure-max-concurrent-sessions
+title: How to configure the maximum concurrent Sessions on the Worker
+sidebar_label: Max concurrent Sessions
+description: Set EnableSessionWorker to true in the Worker options.
+---
+
```

Comment:
Can we add a boilerplate comment to the top of Markdown files that are generated from our docs-as-code files, similar to the "don't edit me" comments that appear at the top of our guides?

Reply:
Added a task for this.